### PR TITLE
Redux/nov5

### PR DESCRIPTION
**Pull request by jperezlatimes**

### CODE REVIEW

File: archive.php [hunk 1]
```diff
@@ -10,61 +10,45 @@
...
-					 * Include the Post-Format-specific template for the content.
-					 * If you want to override this in a child theme, then include a file
-					 * called content-___.php (where ___ is the Post Format name) and that will be used instead.
-					 */
-					//get_template_part( 'template-parts/content', get_post_format() );
-				?>
-
-			<?php endwhile; ?>
+		<main id="main" class="site-main container-fluid" role="main">
+			<section class="hero jumbotron school-archive-header row">
```
Comment (Charlex) [inline archive.php]: jumbotron, love it

File: author.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+			<section class="hero jumbotron archive-header row">
+
+				<div class="container">
+					<div class="author-info">
+						<figure class="author-avatar">
+							<?php echo get_avatar( get_the_author_meta( 'ID' ), 80, '', '', array( 'class' => 'img-circle' ) ); ?>
+						</figure><!-- #author-avatar -->
+						<div class="author-description">
+							<!-- School -->
+							<?php hsinsider_school_link(); ?>
```
Comment (Charlex) [inline author.php]: is `hsinsider_school_link` used anywhere besides the author page? I'm willing to bet its not just a link, but its like a div, heading/description etc.

File: client/js/eu-gallery.js [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+ * Author: James Perez
+ * Created On: 08/19/15
+ */	    
+
+jQuery( document ).ready( function( $ ) { 
+
+	/**
+	 * If a gallery doesn't exist on this page, then don't do anything
+	 */
+	if( $( '.eu_gallery_container' ).length ) {
```
Comment (Charlex) [inline client/js/eu-gallery.js]: we still using the eu prefix?

Comment (Charlex) [inline client/js/eu-gallery.js]: I guess its fine if you're like totally borrowing the gallery code with no modifications from them to keep the namespace eu, but maybe truthfully this should just live as a plugin?

File: author.php (hunk 2 above)
Comment (jperezlatimes) [inline author.php]: Yeah it's used a few times.  More or less wherever there is a link to a school on the site

File: client/js/navigation.js [hunk 4]
```diff
@@ -4,8 +4,6 @@
  * TODO: refactor if time allows
  */
 
-var menuContainer = false;
-var thisClick = false;
```
Comment (Charlex) [inline client/js/navigation.js]: it brings me great relief to see these weird, hard to understand variables that don't look like booleans but really are booleans to go away.

File: client/js/eu-gallery.js (hunk 3 above)
Comment (jperezlatimes) [inline client/js/eu-gallery.js]: It really should, you're right.  Maybe when I'm in a refactoring mood again I'll change it, but it works for now.

File: client/js/navigation.js (hunk 4 above)
Comment (jperezlatimes) [inline client/js/navigation.js]: Me too

File: client/js/responsive-ads.js [hunk 5]
```diff
@@ -15,42 +15,113 @@ googletag.cmd = googletag.cmd || [];
...
 
-	googletag.pubads().setTargeting( 'ptype', ['sf'] );
+	//Adslot 10 declaration
+	gptadslots[14] = googletag.defineSlot( '/4011/trb.latimes/hsinsider', [[300, 250]], 'div-gpt-ad-283030070724299354-2' ).setTargeting( 'pos', ['2'] ).addService( googletag.pubads() );
+
+	//Adslot 11 declaration
+	gptadslots[15] = googletag.defineSlot( '/4011/trb.latimes/hsinsider', [[320, 50]], 'div-gpt-ad-283030070724299354-3' ).setTargeting( 'pos', ['1'] ).addService( googletag.pubads() );
+
+	//Adslot 12 declaration
+	gptadslots[16] = googletag.defineSlot( '/4011/trb.latimes/hsinsider', [[320, 50]], 'div-gpt-ad-283030070724299354-4' ).setTargeting( 'pos', ['2'] ).addService( googletag.pubads() );
```
Comment (Charlex) [inline client/js/responsive-ads.js]: holy balls theres a lot of ad slots. We really should have a plugin UI to manage these

**Comment by Charlex** [comment]
Nice work, go take a break :palm_tree: :palm_tree: :sunny: :shell: :ocean:

**Pull request merged**